types/src/on_chain_config/aptos_features.rs

```diff
@@ -37,9 +37,10 @@ pub enum FeatureFlag {
     EMIT_FEE_STATEMENT = 27,
     STORAGE_DELETION_REFUND = 28,
     SIGNATURE_CHECKER_V2_SCRIPT_FIX = 29,
-    AGGREGATOR_SNAPSHOTS = 30,
+    AGGREGATOR_API = 30,
     SAFER_RESOURCE_GROUPS = 31,
     SAFER_METADATA = 32,
+    AGGREGATOR_EXECUTION = 33,
```

Comment:
AGGREGATOR_EXECUTION is 33 here. In features.move, it's 32. Is that fine?

Reply:
@vusirikala - nice catch, that was a rebase issue.